SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -371,32 +365,31 @@ void ComponentReader::ApplySchemaObject(Schema_Object* ComponentObject, UObject&
 
 					if (bEventTracerEnabled)
 					{
-						PropertySpanIds.Add(Parent.Property, SpanId);
+						ObjectRepNotifiesOut.PropertySpanIds.Add(Parent.Property, SpanId);
```

Comment:
hm, we should maybe get @MatthewSandfordImprobable's opinion or @improbable-danny's opinion on how this will affect event tracing spans and if we are doing things correctly.

Reply:
Don't see any issue here in regards to event tracing. Was there an issue you had in mind or just wanted a check over?

Reply:
Just wanted to check, wasn't sure if we were messing with causality here for the spans, but yeah, probably aren't